tests/telegram/test_examples.py

```diff
@@ -0,0 +1,30 @@
+"""
+This script ensures that example scripts can successfully compile and are ready to run
+"""
+import importlib
+
+import pytest
+
+from tests import utils
+from dff.utils.testing.toy_script import HAPPY_PATH
+from dff.utils.testing.common import check_happy_path
+
+dot_path_to_addon = utils.get_path_from_tests_to_current_dir(__file__, separator=".")
+
+
+def test_unhappy_path(pipeline_instance):
```

Comment:
Why unhappy? :(
Is it used anywhere, actually?..

Reply:
yes,

Reply:
to raise coverage of tests